subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/cpu.h

```diff
@@ -16,12 +16,21 @@ static inline void cpu_sleep(void)
 #endif
 }
 
-static inline void cpu_dsb(void)
+static inline void cpu_dmb(void)
```

Comment:
This is OK for now, but I feel this belongs somewhere in `arch/` ?

Reply:
Will use it, when an interface is available in ARCH, but dont want to go into introducing one now which is going to to be ARM unique.